vault/sessions.go

```diff
@@ -139,7 +139,7 @@ func (s *KeyringSessions) Store(profile string, session sts.Credentials, expires
 		Data:        bytes,
 
 		// specific Keychain settings
-		KeychainNotTrustApplication: true,
+		KeychainNotTrustApplication: false,
```

Comment:
While boolean default value is false, so I believe this line could be omitted, I have left it to be explicit. I am open to removal.

Also, I haven't tested removing it. :)